packages/components/src/Disclosure/Disclosure.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { ReactNode, useState } from "react";
+import classnames from "classnames";
+import styles from "./Disclosure.css";
+import { Heading } from "../Heading";
+import { Icon } from "../Icon";
+
+export interface DisclosureProps {
```

Comment:
Do we need to export `DisclosureProps` here?

Reply:
This one is an odd one... I had to export it otherwise CI was giving me troubles:
https://app.circleci.com/pipelines/github/GetJobber/atlantis/3123/workflows/4fb09704-029b-4cbd-9df2-5819d32a3ad4/jobs/13958

(however local builds & lint worked fine)

Reply:
If you take it out of the exports in index.ts as well does it work?